src/Orchard.Environment.Extensions/ExtensionManager.cs

```diff
@@ -199,8 +199,20 @@ public Task<ExtensionEntry> LoadExtensionAsync(IExtensionInfo extensionInfo)
 
             return loadedFeatures.AsEnumerable();
         }
+        public IEnumerable<IFeatureInfo> GetFeatures()
+        {
+            var allUnorderedFeatures = GetExtensions().Features.ToArray();
```

Comment:
I really meant to remove the .Features on the extension list. It's a very not intuitive.